Doc/using/cmdline.rst

```diff
@@ -1090,13 +1098,28 @@ Debug-mode variables
    If set, Python will dump objects and reference counts still alive after
    shutting down the interpreter.
 
-   Need Python configured with the :option:`--with-trace-refs` build option.
+   Needs Python configured with the :option:`--with-trace-refs` build option.
 
-.. envvar:: PYTHONDUMPREFSFILE=FILENAME
+.. envvar:: PYTHONDUMPREFSFILE
 
    If set, Python will dump objects and reference counts still alive
-   after shutting down the interpreter into a file called *FILENAME*.
+   after shutting down the interpreter into a file under the path given
+   as the value to this environment variable.
 
-   Need Python configured with the :option:`--with-trace-refs` build option.
+   Needs Python configured with the :option:`--with-trace-refs` build option.
 
    .. versionadded:: 3.11
+
+.. envvar:: PYTHON_PRESITE
+
+   If this variable is set to a module, that module will be imported
+   early in the interpreter lifecycle, before ``site.py`` is executed,
```

Comment:
I like this doc the most, you should copy it in other places where you documented the feature :-) For example, the PyConfig doc is different and IMO misleading.

```suggestion
   early in the interpreter lifecycle, before the :mod:`site` module is executed,
```

I suggest to add:

I suggest:

```
It can be used to execute code early during Python initialization.
```

To highlight, that it's not only about *importing* a module, but also to execute it.

If you want, you can add:

```
It can be a sub-module of a package such as `package.module`.
```